src/App.tsx

```diff
@@ -15,35 +30,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type SelectedFilter = 'alphabetically' | 'length' | 'reverse' | '';
+
 export const App: React.FC = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [selecteFilter, setSelectedFilter] = useState<SelectedFilter>('');
```

Comment:
There is a typo in the variable name `selecteFilter`. It should be `selectedFilter` to match the intended usage.